libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectDistanceInfo.kt

```diff
@@ -0,0 +1,44 @@
+package com.mapbox.navigation.base.trip.model.roadobject
+
+/**
+ * RoadObjectDistanceInfo
+ *
+ * @param distanceInfoType type of the distance info object
+ */
+abstract class RoadObjectDistanceInfo(
+    @RoadObjectDistanceInfoType.Type val distanceInfoType: Int,
+) {
+
+    /**
+     * Distance to start of the object
```

Comment:
Is it distance from current location or from the start of the route?

Reply:
from current location. Fixed description